internal/ethapi/simulate.go

```diff
@@ -50,8 +51,8 @@ const (
 
 // simBlock is a batch of calls to be simulated sequentially.
 type simBlock struct {
-	BlockOverrides *BlockOverrides
-	StateOverrides *StateOverride
+	BlockOverrides *override.BlockOverrides
```

Comment:
`override.BlockOverrides` kinda duplicates the "Overrides" part. Maybe we could rename it to `override.Block` and `overrides.State` wdyt? cc @fjl

Reply:
Sure, we could change the name. All of this is in internal/ anyway.

Reply:
I do think that introducing types `Block` and `State` is a bit weird, even though it's technically possible, because these words have a very definitive meaning already. I personally don't mind the duplication in naming for these types, since they are low-use.